pytket_qir/qir.py

```diff
@@ -400,11 +450,82 @@ def condition_zero_block():
                 one=lambda: condition_one_block(),
                 zero=lambda: condition_zero_block(),
             )
+        elif isinstance(op, WASMOp):
+            cregs = _retrieve_registers(circ.bits, BitRegister)
+            args = command.args
+
+            inputs: List[str] = []
+            outputs: List[str] = []
+            for reglist, sizes in [
+                (inputs, op.input_widths),
+                (outputs, op.output_widths),
+            ]:
+                if not sizes:
+                    ValueError("WASM input or output registers have empty widths.")
```

Comment:
Hm, not sure what you mean. The idea here is to check whether there isn't any empty input registers. Although it might be permitted to input empty registers, it feels like there should be a check at some point.

Reply:
Done.